rkt/rkt.go

```diff
@@ -221,7 +228,9 @@ func main() {
 	// cmdRkt.GenBashCompletionFile("dist/bash_completion/rkt.bash")
 	// os.Exit(0)
 
-	cmdRkt.Execute()
+	if err := cmdRkt.Execute(); err != nil && cmdExitCode == 0 {
```

Comment:
We don't need to do anything with the error since cobra will have already logged it. As far as I can tell any errors indicate that cobra did not execute a command so there isn't any false positives. As such the check for `cmdExitCode == 0` is probably not needed but I included it just in case there was ever a code path where cmdExitCode got set to some other code.